src/__experimental__/components/radio-button/radio-button.stories.js

```diff
@@ -87,6 +89,8 @@ const radioComponent = themeName => () => {
         name='frequency'
         legend={ text('groupLabel', 'Please select a frequency from the options below') }
         onChange={ handleGroupChangeFactory(radioToggleGroupStore) }
+        radioButtonInline={ radioButtonInline }
```

Comment:
This looks like an anti-pattern, we're trying to favour composition e.g. 
`<RadioButtonGroup inline legend={<Legend inline />}`

However, it would be difficult to do all of that in this PR, I think we should change the prop name to `inline` this would make our work in the future easier.
```suggestion
        inline={ radioButtonInline }
```